src/mdtf.py

```diff
@@ -84,6 +84,7 @@ def _framework_init(self, code_root, defaults_rel_path):
         self._post_parse_hook(cli_obj, config)
```

Comment:
New PODs shouldn't modify (or need to modify) framework code in `/src`. Please commit the changes from `git checkout develop -- src` to bring the contents of `/src` back to what's in the develop branch.

Reply:
Done. See afb8f0a